corehq/apps/accounting/forms.py

```diff
@@ -852,6 +852,14 @@ class ChangeSubscriptionForm(forms.Form):
         label=gettext_lazy("Edition"), initial=SoftwarePlanEdition.ENTERPRISE,
         choices=SoftwarePlanEdition.CHOICES,
     )
+    new_plan_visibility = forms.ChoiceField(
+        label=gettext_lazy("Visibility"), initial=SoftwarePlanVisibility.PUBLIC,
+        choices=SoftwarePlanVisibility.CHOICES,
+    )
+    most_recent_version = forms.ChoiceField(
+        label=gettext_lazy("Most Recent Version"), initial="True",
```

Comment:
The associated ticket mentions "Show Most Recent Version" or "Show All Versions", which were easier for me to understand than "Most Recent Version". Would it make sense to change the name here? Also, if this will always be a boolean option, a `ChoiceField` doesn't seem like the best fit. Is there anything preventing us from turning this in to a checkbox?

Reply:
<img width="715" alt="image" src="https://github.com/dimagi/commcare-hq/assets/39149002/bead255e-e438-4ffb-9157-f07ca7c9f6fe">
I didn't use "Show Most Recent Version" and "Show All Versions", because I feel, with the bold field "Most Recent Version", it's a bit redundant... But I made the change in c5a94837951e43ccfe0baf8f26eaee3532519918. 
Regarding checkbox, this is just a personal style preference. Since those fields are all filters, I want to them all looks like the same component - a dropdown menu.